src/b2b/organizations_members_oauth_providers.rs

```diff
@@ -0,0 +1,133 @@
+// !!!
+// WARNING: This file is autogenerated
+// Only modify code within MANUAL() sections
+// or your changes may be overwritten later!
+// !!!
+
+use serde::{Deserialize, Serialize};
+
+/// GoogleResponse: Response type for `OAuthProviders.google`.
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct GoogleResponse {
+    /// request_id: Globally unique UUID that is returned with every API call. This value is important to log
+    /// for debugging purposes; we may ask for this value to help identify a specific API call when helping you
+    /// debug an issue.
+    pub request_id: String,
+    /// provider_type: Denotes the OAuth identity provider that the user has authenticated with, e.g. Google,
+    /// Microsoft, GitHub etc.
+    pub provider_type: String,
+    /// provider_subject: The unique identifier for the User within a given OAuth provider. Also commonly called
+    /// the `sub` or "Subject field" in OAuth protocols.
+    pub provider_subject: String,
+    /// access_token: The `access_token` that you may use to access the User's data in the provider's API.
+    pub access_token: String,
+    /// access_token_expires_in: The number of seconds until the access token expires.
+    pub access_token_expires_in: i32,
+    /// id_token: The `id_token` returned by the OAuth provider. ID Tokens are JWTs that contain structured
+    /// information about a user. The exact content of each ID Token varies from provider to provider. ID Tokens
+    /// are returned from OAuth providers that conform to the [OpenID Connect](https://openid.net/foundation/)
+    /// specification, which is based on OAuth.
+    pub id_token: String,
+    /// scopes: The OAuth scopes included for a given provider. See each provider's section above to see which
+    /// scopes are included by default and how to add custom scopes.
+    pub scopes: std::vec::Vec<String>,
+    /// status_code: The HTTP status code of the response. Stytch follows standard HTTP response status code
+    /// patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX
+    /// are server errors.
+    #[serde(with = "http_serde::status_code")]
+    pub status_code: http::StatusCode,
+    /// refresh_token: The `refresh_token` that you may use to obtain a new `access_token` for the User within
+    /// the provider's API.
+    pub refresh_token: std::option::Option<String>,
+}
+
+/// MicrosoftRequest: Request type for `OAuthProviders.google`, `OAuthProviders.microsoft`.
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub struct MicrosoftRequest {
```

Comment:
Request type for `google`? Sus...

Reply:
Thanks for updating codegen to handle this! Looks all better in [7798ba1](https://github.com/stytchauth/stytch-rust/pull/15/commits/7798ba164986803c6e0702267509188d2a88d63e)